native/src/sagas/loadCityContent.ts

```diff
@@ -177,7 +177,7 @@ export default function* loadCityContent(
     }
 
     const { featureFlags } = buildConfig()
-    const [categoriesMap, events, pois] = (yield all([
+    const [categoriesMap, events] = (yield all([
```

Comment:
here i really prefer if you write `_pois`, as we in fact do load the pois and even cast the array type below